atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2380,4 +2381,11 @@ private Meter getMeter(String name, TableReference tableRef) {
                 name,
                 metricsManager.getTableNameTagFor(tableRef));
     }
+
+    private Counter getCounter(String name, TableReference tableRef) {
```

Comment:
`getMeter` should now be unused right? can we get rid of that?

Reply:
yep, removed `getMeter`.